### DISCUSSION TITLE
Google Analytics 4 for Google Tag Manager

### DISCUSSION THREAD

**Discussion by eneskirca** [breezefront/community#100] [3. Integrations]
I am writing to inquire about the compatibility of the "Google Analytics 4 for Google Tag Manager" extension with your platform.

As a business owner, Google Analytics plays a critical role in understanding our website traffic and user behavior. We recently came across the "Google Analytics 4 for Google Tag Manager" extension by MagePal and were impressed with its features and functionality. We believe this extension could greatly benefit our business and help us optimize our website performance.

However, in order to take advantage of this opportunity, we need to ensure that the extension can be easily integrated with your platform. We would appreciate any information or assistance you can provide regarding this matter.

Thank you for your attention to this request, and I look forward to hearing back from you soon.

**Comment by vseager** [comment]
Yes please add support for MagePal GA4 / Tag Manager:

https://github.com/magepal/magento2-google-tag-manager
